.github/workflows/checks.yml

```diff
@@ -12,23 +12,27 @@ on:
       - main
 
 jobs:
-  clang-format:
+  lint:
     runs-on: ubuntu-latest
-    container: ghcr.io/acts-project/format10:v11
+    env:
+      PRE_COMMIT_HOME: '/tmp/pre-commit'
+
     steps:
-      - uses: actions/checkout@v4
-      - name: Check
-        run: .github/check_format.sh .
-      - uses: actions/upload-artifact@v4
```

Comment:
Does this still upload artifacts? That is useful for people who want to contribute fixes

Reply:
It doesn't. However, pre-commit makes it **extremely easy** to apply conformant fixes, so I would steer people towards that.